### PR TITLE
--iptables=false willnot add rules even if --ip-masq=true

### PR DESCRIPTION
**Pull request by bbayani**
### Describe the proposed changes
Updated information for dockerd option --iptables which clarifies if disabled, docker will not add iptable rules even if --ip-masq=true

### Related issue

Fixes https://github.com/docker/docker/issues/28133

### Please take a look
@justincormack

### CODE REVIEW

**Comment by johndmulhausen** [comment]
Deploy preview ready!

Built with commit dc6dcd008c4790de48734841675857d84e4a5c92

https://deploy-preview-565--docker-docs.netlify.com

**Comment by bbayani** [comment]
@justincormack / @mstanleyjones Can you please take a look?

File: engine/reference/commandline/dockerd.md [hunk 1]
```diff
@@ -1272,7 +1272,7 @@ pid file here.
 - `--iptables=false` prevents the Docker daemon from adding iptables rules. If
   multiple daemons manage iptables rules, they may overwrite rules set by
   another daemon. Be aware that disabling this option requires you to manually
-  add iptables rules to expose container ports.
+  add iptables rules to expose container ports. Disabling this also avoids creating nat masquerade rules even if --ip-masq is set to true. This will result in no external connectivity outside of the docker host when using networks other than the default bridge.
```
Comment (mstanleyjones) [inline engine/reference/commandline/dockerd.md]: nit: Please wrap long lines at 80 columns if possible.

Consider simplifying the wording, to something like:

If you prevent Docker from adding `iptables` rules, Docker will also not add IP masquerading rulesr, even if you set `--ip-masq` to `true`. Without IP masquerading rules, no external host will be able to connect to the containers managed by this Docker daemon and Docker containers will not be able to connect to external hosts or the internet.

(I think the last part is true, am I correct?)

**Comment by bbayani** [comment]
Thanks @mstanleyjones. I have updated the commit as per your feedback. Can you please check again?

**Comment by justincormack** [comment]
LGTM

**Comment by bbayani** [comment]
@mstanleyjones can you please take a look?

**Comment by mstanleyjones** [comment]
I just realized these changes should have been made in the `docker/docker` repo. I'll make the `docker/docker` PR on your behalf and ping you there. Sorry for the confusion. This is because these are reference docs that are vendored from that repo.

**Comment by mstanleyjones** [comment]
Closed in favor of https://github.com/docker/docker/pull/28607.

**Comment by Svolentin** [comment]
does anyone have workaround, how to work with:
working nat from inside the containers
working external access to binded port
and working iptables rules which flushed several times a day in prod.
I think docker MUST have generator of iptables rules to the output. Now we cant use docker in production because iptables issue when they flushed, or network access from container does not working, when all other working fine

**Comment by mstanleyjones** [comment]
@Svolentin this kind of question is best asked on the [Docker community slack](https://dockercommunity.slack.com) or on forums.docker.com. Thanks!

**Pull request closed without merging**